packages/installations/src/api/common.ts

```diff
@@ -75,10 +75,30 @@ export function getHeadersWithAuth(
   return headers;
 }
 
-export interface ErrorData {
-  code: number;
-  message: string;
-  status: string;
+export interface ErrorResponse {
+  error: {
+    code: number;
+    message: string;
+    status: string;
+  };
+}
+
+/**
+ * Calls the passed in fetch wrapper and returns the response.
+ * If the returned response has a status of 5xx, re-runs the function once and
+ * returns the response.
+ */
+export async function retryIfServerError(
+  fn: () => Promise<Response>
+): Promise<Response> {
+  const result = await fn();
+
+  if (String(result.status)[0] === '5') {
```

Comment:
That's pretty gross. Why not just check >=500&&<600?

Reply:
Ehh, it's fine either way, but sure.